### ISSUE TITLE
Lua filters: header content is nil in pandoc-2.16?

### ISSUE THREAD

**Issue by wlupton** [jgm/pandoc#7652]
I think that header content is nil in pandoc-2.16? I'm wondering whether this might be related to #7647?

With this input (`header.md`):
```
# Foo
```

and this filter (`header.lua`):
```
function Header(header)
    io.stderr:write(tostring(header) .. '\n')
    io.stderr:write('header ' .. tostring(header.level) .. ' ' ..
                        tostring(header.content) .. '\n')
end
```

I get this with pandoc 2.16:
```
% pandoc header.md -L header.lua 
Header 1 ("foo",[],[]) [Str "Foo"]
header 1 nil
<h1 id="foo">Foo</h1>
```
```
% pandoc --version
pandoc 2.16
```

**Comment by tarleb** [comment]
Yes, this is related in that I somehow missed it and didn't write sufficient tests to catch my error. I added a fix in 759aa5095101351bc2e4d2c4629df945b743e7db. Thanks for the report!

**Comment by wlupton** [comment]
Thanks very much.
